otelcol/command.go

```diff
@@ -52,10 +67,11 @@ func updateSettingsUsingFlags(set *CollectorSettings, flags *flag.FlagSet) error
 	if len(resolverSet.URIs) == 0 {
 		return errors.New("at least one config flag must be provided")
 	}
-	// Provide a default set of providers and converters if none have been specified.
-	// TODO: Remove this after CollectorSettings.ConfigProvider is removed and instead
```

Comment:
It's okay if we know this will be tracked, but I think we should probably leave this todo until we fully remove this check.

Reply:
Added back